### DISCUSSION TITLE
XZ malicious

### DISCUSSION THREAD

**Discussion by arman-maghsoudnia** [aonez/Keka#1527] [Q&A] [answered]
Dear all,

I have heard that one version of XZ turned out to have a backdoor, hence; was malicious. How can I know which version of XZ Keka is using to be sure that it is not the malicious version? 

Best

**Comment by aonez** [comment, marked as answer]
Keka is using version 5.6.2, will be updated to 5.6.3 in the next release. You can always check the version used in Keka - About at.

Keka never used any of the affected versions and anyway due to sandboxing the malicious code will be innocuous. You can check more here: 

https://github.com/aonez/Keka/issues/1425#issuecomment-2028803707
